microsoft-edge/webview2/release-notes/index.md

```diff
@@ -72,6 +72,338 @@ See also:
 <!-- end of template -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2651.64
+
+Release Date: August 7, 2024
+
+[NuGet package for WebView2 SDK 1.0.2651.64](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2651.64)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 127.0.2651.64 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+The following APIs have been promoted to Stable and are now included in this Release SDK.
+
+
+<!-- ---------- -->
+* Updated the WebMessageObjects API to allow injecting DOM objects into WebView2 content that's constructed via the app, and via the `CoreWebView2.PostWebMessage` API in the other direction.  Added a new web object type to represent a file system handle that can be posted to the web content to provide it with filesystem access.
```

Comment:
```suggestion
* Updated the WebMessageObjects API (such as `typeName.memberName` and `typeName.memberName`) to allow injecting DOM objects into WebView2 content that's constructed via the app, and via the `CoreWebView2.PostWebMessage` API in the other direction.  Added a new web object type (`typeName`) to represent a file system handle that can be posted to the web content to provide it with filesystem access.
```
In order to place this list of APIs within "Overview of APIs" such as within section:
Overview of WebView2 features and APIs > Web/native interop > Web messaging
need to specify which APIs are being referred to, in the lead-in text.  Which typeName and memberName is the lead-in text referring to?  
Expect this tab-set and lead-in to be somehow integrated into live article section: 
https://learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis#web-messaging
@victorhuangwq @peiche-jessica

Reply:
my attempted mapping from lead-in text to API list: 
![image](https://github.com/user-attachments/assets/760f33a6-4c4d-4581-9b14-04100a06c034)

Reply:
> Added a new web object type (`FileSystemHandle`) to represent a file system handle that can be posted

Reply:
result:
Added a new web object type (`CoreWebView2FileSystemHandle`) to represent a file system handle that can be posted to the web content to provide it with filesystem access.

Reply:
keeping comment open in case further clarifications